### PR TITLE
Feat/api endpoints

### PR DESCRIPTION
**Pull request by rishusingh022**

### CODE REVIEW

**Review by rohini-senthil — APPROVED** [review]

File: src/constants/apiEndpoints.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+export const DELETE_ENGAGEMENT_DATA_URL = id => ({
+  url: `api/projects/${id}`,
+  method: 'delete',
+});
+
+export const GET_CASE_STUDIES_DATA_URL = {
+  url: 'api/case-studies',
+  method: 'get',
+};
+export const CREATE_CASE_STUDIES__DATA_URL = {
```
Comment (Chetan-MckinseyBEBR) [inline src/constants/apiEndpoints.js:53]: @rishusingh022 why double underscores in naming convention?

File: src/constants/apiEndpoints.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  url: 'api/case-studies',
+  method: 'post',
+};
+export const GET_CASE_STUDIES__DATA_BY_ID_URL = id => ({
```
Comment (Chetan-MckinseyBEBR) [inline src/constants/apiEndpoints.js:57]: add a new line after every function

**Review by Chetan-MckinseyBEBR — COMMENTED** [review]
- There are double underscores in variable/function names.
- Added comment on code format

File: src/constants/apiEndpoints.js (hunk 1 above)
Comment (rishusingh022) [reply src/constants/apiEndpoints.js:53]: Sorry it's a typo I will fix it

File: src/constants/apiEndpoints.js (hunk 2 above)
Comment (rishusingh022) [reply src/constants/apiEndpoints.js:57]: Suree I will add it in the new pull request thanks

**Pull request merged**